lib/libspl/include/sys/types.h

```diff
@@ -74,4 +74,9 @@ typedef union {
 
 #include <sys/param.h> /* for NBBY */
 
+#ifndef _WIN32
+#define	posix_memalign_free free
+#define	NTSTATUS int32_t
```

Comment:
NTSTATUS? Why do we need this for upstream?

Reply:
We don't really, I'll remove it and we can maintain a commit in our own branch.

Reply:
See my other comment. This entire hunk can be dropped.